FND/Places/NorthAmerica/USPostalServiceAddresses.rdf

```diff
@@ -198,7 +198,7 @@
 				<owl:someValuesFrom rdf:resource="&fibo-fnd-plc-uspsa;USPostalServiceAddressIdentifier"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>delivery point code set</rdfs:label>
+		<rdfs:label>delivery address code set</rdfs:label>
```

Comment:
seems a significant change - that has not been justified.

Reply:
The original label was wrong - I just fixed it, that's all.